README.md

```diff
@@ -96,17 +103,103 @@ For example, if you have
 This will be copied to the file `/etc/containers/ansible-kubernetes.d/myappname.yml` on
 the managed node.
 
+### podman_quadlet_specs
+
+List of [Quadlet specifications]
+(https://man.archlinux.org/man/extra/podman/quadlet.5.en)
```

Comment:
Can we point to https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html instead?